crates/services/gas_price_service/src/v1/da_source_service/service.rs

```diff
@@ -126,9 +206,15 @@ where
     }
 }
 
-pub fn new_service<S: DaBlockCostsSource>(
+pub fn new_da_service<S: DaBlockCostsSource>(
```

Comment:
Since we don't set `recorded_height` I assume this function is used only in tests? Should we put in under the `test-helpers` feature?